src/test/scala/dxWDL/dx/DxdaManifestTest.scala

```diff
@@ -55,7 +55,7 @@ class DxdaManifestTest extends FlatSpec with Matchers {
             "folder" -> JsString(local.toFile().getParent())
         )
         JsObject(fields)
-    }.toVector
+    }.toVector.reverse
```

Comment:
Why is the sort order reversed? Does `system/findDataObjects` return objects sorted by a different field than `system/describeDataObjects`?

Reply:
I am not sure according to which criteria it returns the files, but they seem to be reversed.
I add few more testcases for bulk describe.

Is it ordering of elements in returned map relevant?

Reply:
I don't think it's relevant, no. Was just wondering where it comes from.